### DISCUSSION TITLE
Example PDF archive

### DISCUSSION THREAD

**Discussion by EliotJones** [UglyToad/PdfPig#302] [General]
Since it is useful to have many PDF documents when carrying out document layout analysis, new feature development and performance work I'm sharing the test archive I put together during the initial development of the library.

https://drive.google.com/file/d/1C6bD4BVIc4pxT4oDEmUT1jLV_oi9NYUh/view?usp=sharing

These PDFs are useful for testing bugfixes and running profiling since they represent a broad cross-section of producers and document types.

**Comment by fnatzke** [comment]
In #532 @BobLd provide an additional PDF repository here: 

https://github.com/pdf-association/pdf-corpora#safedocs-issue-tracker-corpus

**Comment by fnatzke** [comment]
The attach zip contains the URLs of of an additional 30,000+ PDFs (~50GB) are available.

To download (unix *nix or windows subsystem for linux (WSL)) enter:

cut -d, -f1 URLsofExamplePDFs.txt | wget -i -



[URLsofExamplePDFs.zip](https://github.com/UglyToad/PdfPig/files/11035338/URLsofExamplePDFs.zip)
